src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberContainerTypeSymbol.vb

```diff
@@ -1378,9 +1378,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Return _lazyLexicalSortKey
         End Function
 
+        Public Function LocationKey(ByVal l As Location) As Long
+            If Not l.IsInSource Then
+                Dim vbLocation = TryCast(l, VBLocation)
+                Return 1L << 62 + If(vbLocation Is Nothing, 0, vbLocation.PossiblyEmbeddedOrMySourceSpan.Start)
+            End If
+
+            Dim treeIndex = Me.DeclaringCompilation.GetSyntaxTreeOrdinal(l.SourceTree)
```

Comment:
Consider spelling out the type in order to improve readability of the code. I believe the general rule is - the type must be named unless it is named in the initializer.